Main/src/main/java/com/main/acad/controller/UsersConntroller.java

```diff
@@ -23,7 +23,7 @@ public void createNewUser(HttpServletRequest request, HttpServletResponse respon
                 Integer.valueOf(request.getParameter("password")), request.getParameter("role"));
         try {
             response.getWriter().write(String.valueOf(result));
-        } catch (IOException | ConnectionPoolFailedException| UserDaoFailedException e) {
+        } catch (IOException | ConnectionPoolFailedException | UserDaoFailedException e) {
             logger.info("An error occurred in the ChaptersController in the createNewUser method" + e.getMessage());
             throw new ControllerNotFoundException(e.getMessage());
```

Comment:
it is no "controller not found" situation - you can not throw this exception as it leads to misunderstanding what's really going on, it should be another one with CLEAR and UNDERSTANDABLE and SUITABLE for this situation name

Reply:
do I need to throw the user on the error page???